### PR TITLE
[Display_02] Display2 class uml diagram

### PR DESCRIPTION
**Pull request by jcortes18**

### CODE REVIEW

File: docs/uml-class-diagrams/display02/production/DisplayComm2WayInterfaceImplementation/DisplayComm2WayInterfaceImplementation.puml [hunk 1]
```diff
@@ -4,10 +4,11 @@ package edu.oakland.production.display02 {
     class DisplayComm2WayInterfaceImplementation {
         __
         .. Use Case 1 ..
+        {abstract} + LocationDataPoint passRfidRequest(int rfidNum)
```
Comment (andrewdimmer) [inline docs/uml-class-diagrams/display02/production/DisplayComm2WayInterfaceImplementation/DisplayComm2WayInterfaceImplementation.puml:7]: Can you please move this from the class `DisplayComm2WayInterfaceImplementation` to the interface  `DisplayComm2WayInterface`? Thank you!

**Review by andrewdimmer — CHANGES_REQUESTED** [review]

File: docs/uml-class-diagrams/display02/production/DisplayComm2WayInterfaceImplementation/DisplayComm2WayInterfaceImplementation.puml (hunk 1 above)
Comment (jcortes18) [reply docs/uml-class-diagrams/display02/production/DisplayComm2WayInterfaceImplementation/DisplayComm2WayInterfaceImplementation.puml:7]: moved to DisplayComm2WayInterface. Thanks!

**Review by andrewdimmer — APPROVED** [review]
Looks good to me! Just go ahead and add another reviewer via the top of the right-hand sidebar, then you should be good to merge!

**Review by sydneyhill3901 — APPROVED** [review]
Looks good, no suggestions from me. Approved.

**Pull request merged**